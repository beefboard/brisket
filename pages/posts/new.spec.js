import VueTestUtils from '@vue/test-utils'
import { shallowMount } from '@vue/test-utils'
import newPage from './new.vue'

describe('posts/new.vue', () => {
  it('renders properly', () => {})
})
