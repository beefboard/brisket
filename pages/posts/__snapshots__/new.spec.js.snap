// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`posts/new.vue renders properly 1`] = `
<div class="page-container">
  <div class="form">
    <div>
      <!---->
      <div class="top"><input type="text" placeholder="Title" class="title"> <input id="files" type="file" multiple="multiple" class="files"> <button class="beefbutton upload-button">Attach photo(s)</button></div> <input min-height="300" placeholder="Type something here..." class="content" style="outline: none; font-size: 1rem;"> <button disabled="disabled" class="beefbutton submit-button">Submit</button>
    </div>
  </div>
  <div></div>
</div>
`;
